### DISCUSSION TITLE
Copilot is not Immediately Purchased

### DISCUSSION THREAD

**Discussion by hyperforest** [community/community#51630] [Copilot Conversations]
### Select Topic Area

Question

### Body

I submitted my GitHub Copilot purchase on the link below at 1 Apr
https://github.com/github-copilot/signup/billing?payment_duration=monthly

But when I open my VS code, it shows this error
```
[INFO] [auth] [2023-04-01T03:46:49.144Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-01T03:46:49.145Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."
```

It turns out that on the link above, my next payment is in Apr 30, 2023 (30 days after the subscription submitted, not immediately in the date I click "submit" button on the link above).
![image](https://user-images.githubusercontent.com/28144147/229265316-7a773d58-8827-4e59-aa7e-ebeab6686c06.png)

Questions:
1) How can I get the subscriptions immediately?
2) Am I at the correct link to submit a purchase/subscriptions?

To add some context:
1. I have been purchased a free trial of Copilot before
2. My debit card is *not* declined

Thank you

[EDIT 2/4] Move images, add 2nd question

**Comment by akosiyawin** [comment]
Experiencing the same issue since yesterday, any updates to this?

**Reply by hyperforest** [reply]
Still no. It gets much worse actually: the payment is always 30 days after the day you submit the subscriptions.

**Comment by keyyuki** [comment]
me too. Is this joke of 1/4

**Comment by imronreviady** [comment]
have the same issue and start installing competitor services now

**Comment by hyperforest** [comment]
[SOLVED] by changing the card. Somehow the card I used previously is declined without any alert whatsoever. Discussion closed.
